apollo-core/src/main/java/com/ctrip/framework/foundation/internals/NetworkInterfaceManager.java

```diff
@@ -98,7 +99,19 @@ private void load() {
 
     try {
       Enumeration<NetworkInterface> interfaces = NetworkInterface.getNetworkInterfaces();
-      List<NetworkInterface> nis = interfaces == null ? Collections.<NetworkInterface>emptyList() : Collections.list(interfaces);
+      List<NetworkInterface> nis = interfaces == null ? Collections.<NetworkInterface>emptyList()
+          : Collections.list(interfaces);
+      //sort the network interfaces according to the index asc
+      Collections.sort(nis, new Comparator<NetworkInterface>() {
```

Comment:
can be replaced with `list.sort`?

Reply:
can be replaced with lambda?

Reply:
of course!  but apollo-core is <java.version>1.7</java.version> in pom.xml.
Consider compatibility，i don't use lambda.

Reply:
Got it